packages/babel-preset-env/test/debug-fixtures/android/stdout.txt

```diff
@@ -133,4 +133,4 @@ Using polyfills with `entry` option:
   web.timers { "android":"4" }
   web.immediate { "android":"4" }
   web.dom.iterable { "android":"4" }
-src/in.js -> lib/in.js
\ No newline at end of file
+🎉  Successfully compiled 1 file with Babel.
```

Comment:
Also, all tests in `babel-preset-env` cover default quiet mode.

Reply:
👍